emsdk_manifest.json

```diff
@@ -34,8 +34,8 @@
     "version": "%releases-tag%",
     "bitness": 64,
     "arch": "x86_64",
-    "linux_url": "https://storage.googleapis.com/webassembly/emscripten-releases-builds/linux/%releases-tag%/wasm-binaries.tbz2",
-    "macos_url": "https://storage.googleapis.com/webassembly/emscripten-releases-builds/mac/%releases-tag%/wasm-binaries.tbz2",
+    "linux_url": "https://storage.googleapis.com/webassembly/emscripten-releases-builds/linux/%releases-tag%/wasm-binaries.tar.xz",
```

Comment:
I wonder if it's worth leaving a comment here that the .tbz2 equivalents to these URLs are also tried if xz is not fuond.

Reply:
json doesn't support comments :sob:

Reply:
Oh right, I always for get that 😿